apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -339,6 +501,36 @@ parse_vtep_mac(MAC) ->
         end,
         MACComponents).
 
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_counter:new([
+        {registry, overlay},
+        {name, polls_total},
+        {help, "Total number of times local Mesos overlays have beed polled."}]),
+    prometheus_histogram:new([
+        {registry, overlay},
+        {name, poll_duration_seconds},
+        {buckets, [0.001, 0.01, 0.1, 1.0, 10.0, 60.0]},
+        {duration_unit, seconds},
+        {help, "The time spent polling local Mesos overlays."}]),
+    prometheus_counter:new([
+        {registry, overlay},
+        {name, poll_errors_total},
+        {help, "Total number of errors while polling local Mesos overlays."}]),
+    prometheus_gauge:new([
+        {registry, overlay},
+        {name, networks_configured},
+        {help, "The number of overlay networks configured."}]),
+    prometheus_counter:new([
+        {registry, overlay},
+        {name, network_configuration_failures_total},
```

Comment:
follow-up: https://github.com/dcos/dcos-net/pull/161#discussion_r304359807

I'd say we can just crash the process if there is a lashup error, lashup has its own metrics. Let's do that and rename the metric to `vtep_configuration_failures_total`?

Reply:
I think that whether Lashup has its own metrics or not, should not affect what metrics are needed in this module, cause Lashup is a DB and all, whereas this module uses different terminology and operates with different beasts.